production/loki-mixin-compiled-ssd/dashboards/loki-bloom-gateway.json

```diff
@@ -737,44 +714,941 @@
+                           "pointSize": 5,
+                           "scaleDistribution": {
+                              "type": "linear"
+                           },
+                           "showPoints": "auto",
+                           "spanNulls": false,
+                           "stacking": {
+                              "group": "A",
+                              "mode": "none"
+                           },
+                           "thresholdsStyle": {
+                              "mode": "off"
+                           }
+                        },
+                        "mappings": [ ],
+                        "max": 1,
+                        "min": 0,
+                        "thresholds": {
+                           "mode": "absolute",
+                           "steps": [
+                              {
+                                 "color": "green",
+                                 "value": null
+                              },
+                              {
+                                 "color": "red",
+                                 "value": 80
+                              }
+                           ]
+                        },
+                        "unit": "percentunit"
+                     },
+                     "overrides": [ ]
+                  },
+                  "gridPos": {
+                     "h": 7,
+                     "w": 12,
+                     "x": 12,
+                     "y": 1118
+                  },
+                  "id": 108,
+                  "options": {
+                     "legend": {
+                        "calcs": [ ],
+                        "displayMode": "list",
+                        "placement": "bottom",
+                        "showLegend": true
+                     },
+                     "tooltip": {
+                        "mode": "single",
+                        "sort": "none"
+                     }
+                  },
+                  "panels": [ ],
+                  "pluginVersion": "11.4.0-77663",
+                  "targets": [
+                     {
+                        "datasource": {
+                           "type": "prometheus",
+                           "uid": "000000134"
+                        },
+                        "editorMode": "code",
+                        "expr": "sum(rate(loki_bloom_recorder_series_total{cluster=\"$cluster\", job=\"$namespace/bloom-gateway\", type=\"filtered\"}[$__rate_interval])) by (type)\n/ on () group_left\nsum(rate(loki_bloom_recorder_series_total{cluster=\"$cluster\", job=\"$namespace/bloom-gateway\", type=\"found\"}[$__rate_interval]))",
+                        "hide": false,
+                        "instant": false,
+                        "legendFormat": "__auto",
+                        "range": true,
+                        "refId": "B"
+                     }
+                  ],
+                  "title": "Filtered series",
```

Comment:
This panel was blank for me, while "Found/Skipped/Missing series" wasn't. Bug?

Reply:
This seems to be a bug in Loki where we don't record series any more:

```
sum(rate(loki_bloom_recorder_series_total{cluster="$cluster", job="$namespace/bloom-gateway", type="filtered"}[$__rate_interval]))
```
Does not yield anything, where else
```
sum(rate(loki_bloom_recorder_chunks_total{cluster="$cluster", job="$namespace/bloom-gateway", type="filtered"}[$__rate_interval]))
```
does